src/app/app.component.ts

```diff
@@ -113,7 +113,7 @@ export class AppComponent implements OnInit, OnDestroy {
     analyticsConfig.liveAnalytics = config.liveAnalytics;
     analyticsConfig.showNavBar = !this.hosted;
     analyticsConfig.isHosted = this.hosted;
-    analyticsConfig.permissions = config.permissions || {};
+    analyticsConfig.permissions = config.permissions || { enableLiveViews: true };
```

Comment:
@diamond-darrell - default should be false, just in case

Reply:
it's a leftover, fixed